ts/types/Reaction.ts

```diff
@@ -0,0 +1,147 @@
+import { EmojiSet } from 'emoji-mart';
+
+export class RecentReactions {
+  public items: Array<string> = [];
+  public limit: number = 6;
```

Comment:
this could be a constant outside of the class if you don't need to call it outside.
If you need it in the class but it doesn't change mark it readonly

Reply:
Nice catch, I have moved it out as a const